funcx_endpoint/tests/unit/test_endpoint_unit.py

```diff
@@ -157,7 +157,9 @@ def test_register_endpoint_invalid_response(
 
     register_endpoint_response(endpoint_id=other_endpoint_id)
     with pytest.raises(SystemExit) as pytest_exc:
-        ep.start_endpoint(ep_dir, endpoint_uuid, ep_conf, log_to_console, no_color)
+        ep.start_endpoint(
+            ep_dir, endpoint_uuid, ep_conf, log_to_console, no_color, reg_info={}
```

Comment:
Re: the testing for more than just endpoint_id field.  Just testing for empty {}?  That gets past the validation in _do_start_endpoint but gets to start_endpoint.  Even if you add additional validation, just checking one in test is enough (even if empty), now that I think about it.  (ie this is fine)